### PR TITLE
[1394][IMP]base_ir_values_security_adj: pre-commit, README

### PR DESCRIPTION
**Pull request by Ryoko04**
Task [1394](https://www.quartile.co/web#id=1394&action=771&model=project.task&view_type=form&menu_id=505)

No changes.

### CODE REVIEW

**Review by yostashiro — COMMENTED** [review]
@Ryoko04 Please update the README to briefly explain WHY we need the changes done by this module.

**Comment by Ryoko04** [comment]
@yostashiro I added the description. Can you review it again?

File: base_ir_values_security_adj/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -1,2 +1,2 @@
 This modules does the following:
-  * Add "Employee" group to the "Defaults: alter personal values only" (XML ID: base.ir_values_default_rule)
+  * Add "Employee" group to the "Defaults: alter personal values only" (XML ID: base.ir_values_default_rule)  to restrict "Employee" group user to set default value for "Only you"
```
Comment (yostashiro) [inline base_ir_values_security_adj/readme/DESCRIPTION.rst]:
```suggestion
  * Adds "Employee" group to the "Defaults: alter personal values only" (XML ID: base.ir_values_default_rule).
  * Adds a new record rule "Defaults: alter all values" to which "Admin / Settings" group is assigned.

Without these adjustments, no one would be able to create user-defined defaults that were applicable for all users.
```

Let's try and address the background why we had to make these changes.

**Review by yostashiro — CHANGES_REQUESTED** [review]

File: base_ir_values_security_adj/__manifest__.py [hunk 2]
```diff
@@ -2,18 +2,14 @@
...
-    'installable': True,
+    "name": "Base Ir Values Security Adjustment",
+    "version": "10.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Quartile Limited",
+    "website": "http://www.quartile.co/",
+    "category": "Tools",
+    "depends": ["base"],
+    "data": ["security/base_security.xml"],
+    "application": True,
```
Comment (yostashiro) [inline base_ir_values_security_adj/__manifest__.py]:
```suggestion
```
Sorry, missed this one. We don't need this line. This module is not an "app."

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Review by yostashiro — APPROVED** [review]
LGTM 👍

**Pull request merged**